tests/unit/test_units.py

```diff
@@ -149,32 +154,32 @@ def test_a():
 
 def test_context():
     CO2 = unit_registry("CO2")
-    N = unit_registry("N")
+    N2ON = unit_registry("N2ON")
     with unit_registry.context("AR4GWP100"):
-        np.testing.assert_allclose(CO2.to("N").magnitude, 14 / (44 * 298))
-        np.testing.assert_allclose(N.to("CO2").magnitude, 44 * 298 / 14)
+        np.testing.assert_allclose(CO2.to("N2ON").magnitude, 28 / (44 * 298))
+        np.testing.assert_allclose(N2ON.to("CO2").magnitude, 44 * 298 / 28)
```

Comment:
Can you add a test for N2O <-> CO2 to show that N2O and N2ON result in different values

Reply:
Done